CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Set the path of development pod groups to the top-most shared directory  
```

Comment:
nit I know some don't and its unintuitive a bit but I still update my CHANGELOG entry to point to the PR even without an issue.

Reply:
nah agreed, I prefer to do it too. will add